src/modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,8 +1,163 @@
+      variant="body2"
+      color="text.secondary"
+      align="center"
+      {...props}
+    >
+      {"Copyright © "}
+      <Link href="https://mui.com/">Your Website</Link>{" "}
+      {new Date().getFullYear()}
+      {"."}
+    </Typography>
+  );
+}
+
+// TODO remove, this demo shouldn't need to reset the theme.
+const defaultTheme = createTheme();
 
 const ForgetPass = () => {
+
+  const [loadingBtn, setLoadingBtn] = useState(false);
+  const navigate = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormValuesForgetPass>();
+
+  const backgroundStyle = {
+    backgroundSize: "cover",
+    backgroundPosition: "center",
+    backgroundRepeat: "no-repeat",
+    display: { xs: "none", sm: "flex" },
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    alignItems: "center",
+  };
+
+  const onSubmit: SubmitHandler<FormValuesForgetPass> = async (data) => {
+    // setLoadingBtn(true);
+    try {
+      const response = await axiosInstance.post("admin/users/forgot-password", data);
+      console.log(response);
+      toast.success(response.data.message ||"check your email");
+      navigate("/reset-pass");
+    } catch (error : any) {
+      console.log(error);
+      toast.error(error.response.data.message);
+    } 
+    console.log(data);
+  };
+
   return (
-    <div>ForgetPass</div>
+    <>
+    <ThemeProvider theme={defaultTheme}>
+      <Grid container component="main" sx={{ height: "100vh" }}>
+        <CssBaseline />
+        <Grid item xs={12} sm={6} md={6} component={Paper} elevation={6} square>
+          <h1 className="p-5 text-xl font-serif font-bold">
+            <span className="text-[blue]">Stay</span>cation.
+          </h1>
+          <Box
+            className="px-[0.5em] md:px-[0.5em] lg:px-[5em]"
+            sx={{
+              my: 8,
+              mx: 8,
+              display: "flex",
+              flexDirection: "column",
+              justifyContent: "center",
+            }}
+          >
+            <Typography component="h1" variant="h5">
+              Forgot password
+            </Typography>
+            <p className="mt-4">If you already have an account register <br /> You can <span className="text-[red]">Login here !</span></p>
+            <Box
+              component="form"
+              noValidate
+              onSubmit={handleSubmit(onSubmit)}
+              sx={{ mt: 1 }}
+            >
+              <InputLabel htmlFor="outlined-adornment-email" className="mt-5 my-1">
+                Email
+              </InputLabel>
+              <OutlinedInput
+                id="outlined-adornment-email"
+                fullWidth
+                type="text"
+                label="email"
+                placeholder="Please Type Here"
+                className="bg-[#F5F6F8] mb-3"
+                {...register(
+                  "email",emailValidation
+                )}
+              />
+              {errors.email && (
```

Comment:
we might use the helper text to display the error message and give the state to the error prop, this will change the color of the outline, text and helper text to red.